spec/default_spec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'spec_helper'
+
+describe 'osl-varnish::default' do
+  ALL_PLATFORMS.each do |p|
+    context "#{p[:platform]} #{p[:version]}" do
+      cached(:chef_run) do
+        ChefSpec::SoloRunner.new(p).converge(described_recipe)
+      end
+      it 'converges successfully' do
+        expect { chef_run }.to_not raise_error
+      end
```

Comment:
Since it's a small, one recipe cookbook, adding all the tests here may be a good idea. Thoughts @ramereth ?